src/components/organisms/PaymentModal/PaymentForm.tsx

```diff
@@ -61,32 +63,63 @@ const PaymentForm: React.FunctionComponent<PropsType> = ({
     }
   }, [event.id, venueId, ticketPrice, user, selectedTab]);
 
-  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+  const handleSubmit = async (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    setIsLoading: (isLoading: boolean) => void,
+    operations: () => Promise<void>
+  ) => {
     if (!user) return;
-    setIsFormBeingSubmitted(true);
+    setIsLoading(true);
     e.preventDefault();
     if (!stripe || !elements) {
       setErrorMessage("Oops something wrong happened");
+      setIsPaymentProceeding(false);
```

Comment:
setIsLoading and setIsPaymentProceeding look like they're the same thing. Remove one of them

Reply:
they're not the same. I need to know where to put the loader
<img width="1440" alt="Screen Shot 2020-07-30 at 12 42 43" src="https://user-images.githubusercontent.com/13795384/88914100-65840400-d262-11ea-8ca4-4163ad1a0863.png">

here there is a mistake, I should have written setIsLoading